cmd/crio-status/main.go

```diff
@@ -106,29 +118,12 @@ func containers(c *cli.Context) error {
 		return err
 	}
 
-	fmt.Printf("name: %s\n", info.Name)
-	fmt.Printf("pid: %d\n", info.Pid)
-	fmt.Printf("image: %s\n", info.Image)
-	fmt.Printf("image ref: %s\n", info.ImageRef)
-	fmt.Printf("created: %v\n", info.CreatedTime)
-	fmt.Printf("labels:\n")
-	for k, v := range info.Labels {
-		fmt.Printf("  %s: %s\n", k, v)
-	}
-	fmt.Printf("annotations:\n")
-	for k, v := range info.Annotations {
-		fmt.Printf("  %s: %s\n", k, v)
+	if output, e := genOutput(info); e == nil {
+		fmt.Print(output)
+		return nil
+	} else {
```

Comment:
nit, typically, if we process an error, we would use that block to do so, like:
```
output, err := genOutput(info)
if err != nil {
    return e
}
fmt.Println(output)
```

mostly a style thing, but I would prefer it if you wouldn't mind